src/application/environment.rs

```diff
@@ -0,0 +1,71 @@
+
+#[derive(Debug, Clone)]
+pub struct EffiEnvironment {
+    pub todo_directory_path: String,
```

Comment:
To be consistent, the struct fields should be named according to env variables: `todo_dir`, `todo_file` and `done_file`.

Reply:
Renamed these fields according to the env variable names